components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -893,12 +893,22 @@ bool ConversationDriver::HasPendingConversationEntry() {
   return pending_conversation_entry_ != nullptr;
 }
 
-bool ConversationDriver::IsPageContentsTruncated() {
+int ConversationDriver::GetContentUsedPercentage() {
   if (article_text_.empty()) {
-    return false;
+    return 0;
+  }
```

Comment:
Do we still need this part? Why not just remove it and so this condition will return `100`. Just want to make sure we don't introduce bugs where we show a warning that 0% of the content was used.

Reply:
removed